### PR TITLE
revise manual pages

### PR DESCRIPTION
**Pull request by tnagler**

### CODE REVIEW

File: R/rvine_structure.R [hunk 1]
```diff
@@ -8,34 +8,60 @@
...
+#' [as_rvine_structure()].
 #'
 #' Denoting by `M[i, j]` the array entry in row `i` and column `j` (the
-#' pair-copula index for edge `e` in tree `t` of a `d` dimensional vine is `(M[d
-#' + 1 - e, e], M[t, e]; M[t - 1, e], ..., M[1, e])`. Less formally, 1. Start
-#' with the counter-diagonal element of column `e` (first conditioned variable).
-#' 2. Jump up to the element in row `t` (second conditioned variable). 3. Gather
-#' all entries further up in column `e` (conditioning set).
+#' pair-copula index for edge `e` in tree `t` of a `d` dimensional vine is
+#' `(M[d + 1 - e, e], M[t, e]; M[t - 1, e], ..., M[1, e])`. Less formally, 1.
```
Comment (tvatter) [inline R/rvine_structure.R]: Shouldn't this be a list?

File: R/rvine_structure.R [hunk 2]
```diff
@@ -45,9 +71,10 @@
 #' anti-diagonal entry of a column must not be contained in any column further
 #' to the right. 5. The entries of a column must be contained in all columns to
```
Comment (tvatter) [inline R/rvine_structure.R]: Shouldn't this be a list?

Comment (tnagler) [reply R/rvine_structure.R]: Yep

File: R/rvine_structure.R (hunk 1 above)
Comment (tnagler) [reply R/rvine_structure.R]: Yep

**Pull request merged**